src/basic/in-addr-util.c

```diff
@@ -68,6 +68,15 @@ bool in4_addr_is_localhost(const struct in_addr *a) {
         return (be32toh(a->s_addr) & UINT32_C(0xFF000000)) == UINT32_C(127) << 24;
 }
 
+bool in4_addr_is_not_local_scope(const struct in_addr *a)
+{
```

Comment:
Coding style issue.
Also, I prefer to not include negation in the function name.

Reply:
Sure. Renamed from `in4_addr_is_not_local_scope()` to `in4_addr_is_non_local()`.

I dropped the "scope" part, because that sounds like one of the values from `iproute2`'s `/etc/iproute2/rt_scopes` (which is something different).

Also, I replaced "not" with "non", because it's `is(non-local)` and not `is-not(local)`. I guess, the new name isn't much better though.

Could you suggest a name that is preferable to you? Thanks.

Reply:
how about `in4_addr_is_suitable_for_dnsntp()`? That focuses less on what the function does, but more on the purpose where the function is used. Usually, I think that's not desirable, but in this case it might be. Also, it doesn't have a negation in the name.

Reply:
OK. `in4_addr_is_suitable_for_dnsntp()` seems good, and the function has comments about what it does.

Reply:
Done.